tests/test_accumulator_mean_stack.py

```diff
@@ -97,147 +97,168 @@ def make_mask_map(self, stats_ctrl):
 
     def test_online_coadd_input_variance_true(self):
         """Test online coaddition with calcErrorFromInputVariance=True."""
-        exposures, weights = self.make_test_images_to_coadd()
-        coadd_exposure = self.make_coadd_exposure(exposures[0])
-        stats_ctrl = self.make_stats_ctrl()
-        stats_ctrl.setCalcErrorFromInputVariance(True)
-        mask_map = self.make_mask_map(stats_ctrl)
-
-        stats_flags = afwMath.stringToStatisticsProperty("MEAN")
-        clipped = afwImage.Mask.getPlaneBitMask("CLIPPED")
-
-        masked_image_list = [exp.maskedImage for exp in exposures]
-
-        afw_masked_image = afwMath.statisticsStack(masked_image_list,
-                                                   stats_flags,
-                                                   stats_ctrl,
-                                                   weights,
-                                                   clipped,
-                                                   mask_map)
-
-        mask_threshold_dict = AccumulatorMeanStack.stats_ctrl_to_threshold_dict(stats_ctrl)
-
-        # Make the stack with the online accumulator
-        stacker = AccumulatorMeanStack(
-            coadd_exposure.image.array.shape,
-            stats_ctrl.getAndMask(),
-            mask_threshold_dict=mask_threshold_dict,
-            mask_map=mask_map,
-            no_good_pixels_mask=stats_ctrl.getNoGoodPixelsMask(),
-            calc_error_from_input_variance=stats_ctrl.getCalcErrorFromInputVariance(),
-            compute_n_image=True)
-
-        for exposure, weight in zip(exposures, weights):
-            stacker.add_masked_image(exposure.maskedImage, weight=weight)
-
-        stacker.fill_stacked_masked_image(coadd_exposure.maskedImage)
-
-        online_masked_image = coadd_exposure.maskedImage
-
-        # The coadds match at the <1e-5 level.
-        testing.assert_array_almost_equal(online_masked_image.image.array,
-                                          afw_masked_image.image.array, decimal=5)
-        testing.assert_array_almost_equal(online_masked_image.variance.array,
-                                          afw_masked_image.variance.array)
-        testing.assert_array_equal(online_masked_image.mask.array,
-                                   afw_masked_image.mask.array)
+        for use_weight_arrays in [False, True]:
```

Comment:
Here and elsewhere, I'd recommend making `use_weight_arrays` an argument to the test function and use `lsst.utils.tests.methodParameters`.